zerver/openapi/zulip.yaml

```diff
@@ -1798,6 +1798,53 @@ paths:
                                   "upload_space_used": 0,
                                   "id": 0,
                                 }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to a user's clients when registration of their
+                                push device to bouncer service either succeeds or fails.
+
+                                Helps clients to live-update the `push_devices` dictionary
+                                returned in [`POST /register`](/api/register-queue) response.
+
+                                **Changes**: New in Zulip 11.0 (feature level ZF-1653f2).
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - push_device
+                                op:
+                                  type: string
+                                  enum:
+                                    - update
+                                push_account_id:
+                                  type: string
+                                  description: |
+                                    The push account ID whose registration to bouncer succeed or fail.
+                                status:
+                                  type: string
+                                  description: |
+                                    The updated registration status.
+                                    Either `"active"` or `"failed"`.
+                                error_code:
+                                  type: string
+                                  nullable: true
+                                  description: |
+                                    The error code which resulted in registration to bouncer failure.
+                                    Either `"INVALID_BOUNCER_PUBLIC_KEY"`, `"BAD_REQUEST"`, or `"REQUEST_EXPIRED_ERROR"`.
```

Comment:
I think `REQUEST_EXPIRED` is correct, without `_ERROR`; the type for all these codes is an error, so we don't need to repeat that in the name.

Reply:
Made that change